NOTICE-binary

```diff
@@ -189,42 +189,6 @@ JUnit (4.12)
 
 * License: Eclipse Public License
 
-Hive Beeline
```

Comment:
is it leaked before this pr ?

Reply:
should be retained, the following jars were shipped under `beeline-jars`
```
hive-beeline-3.1.3.jar
hive-common-3.1.3.jar
hive-jdbc-3.1.3.jar
hive-service-3.1.3.jar
hive-service-rpc-3.1.3.jar
```